kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
Right I'm *sure* we had this previously because there were issues we couldn't handle without it.  @abellotti ?